spec/requests/api/v1/sessions/facebook_spec.rb

```diff
@@ -57,12 +57,12 @@
     end
 
     it 'returns status code 403' do
-      expect(response).to have_http_status(403)
+      expect(response).to have_http_status(:forbidden)
     end
     
     it 'returns a not authorized error message' do
       expect(json['error'])
-        .to match('Not Authorized.')
+        .to match('Not authorized.')
```

Comment:
You can match this with the translation that you edited above (a few files above)